pol-core/pol/module/uomod.cpp

```diff
@@ -4471,6 +4477,68 @@ BObjectImp* UOExecutorModule::mf_GetStandingLayers( /* x, y, flags, realm */ )
     return new BError( "Invalid parameter type" );
 }
 
+BObjectImp* UOExecutorModule::mf_GetStandingCoordinates(/* x, y, radius, minz, maxz, realm := _DEFAULT_REALM, movemode := "L", doors_block = 0 */)
+{
+  int dummy, x, y, r, minz, maxz;
+  const String* realmname;
+  const String* movemodename;
+  bool doors_block;
+
+  if (!(
+           getParam(0, dummy) && //FIXME: remove this dummy once you figure out why getting the 0th param fails every time
```

Comment:
There is a bizarre bug here that I could not track down. The first param given to this function would always have the value `0`. No matter what is passed as the value here, `dummy` will always be `0`. This will obviously need to be fixed before merging, but I have no idea what's going wrong :-P